crates/bevy_pbr/src/light/mod.rs

```diff
@@ -496,12 +496,19 @@ pub enum ShadowFilteringMethod {
     Temporal,
 }
 
+/// System sets used to run light-related systems.
 #[derive(Debug, Hash, PartialEq, Eq, Clone, SystemSet)]
 pub enum SimulationLightSystems {
     AddClusters,
     AssignLightsToClusters,
+    /// System order ambiguities between systems in this set are ignored:
```

Comment:
These kinds of comments seem a bit wrong on pub APIs, as a user doesn't need to know that. Maybe we should move these to the ambiguity allowing function calls?

Reply:
Ah, fair point.